src/status_im/utils/ethereum/eip55.cljs

```diff
@@ -9,14 +9,4 @@
 
 (defn valid-address-checksum? [address]
   "verify address checksum according to EIP 55"
-  (let [adHash (ethereum/naked-address (ethereum/sha3
-                                        (string/lower-case (ethereum/naked-address address))))]
-    (every? true?
-            (map-indexed (fn [idx char]
-                           (if (> (compare char "9") 0)
-                             (if (>= (js/parseInt (nth adHash idx) 16) 8)
-                                ;  If true should be upper case
-                               (<= (compare char "Z") 0)
-                                ; If not should be lower case
-                               (> (compare char "Z") 0))
-                             true)) (ethereum/naked-address address)))))
\ No newline at end of file
+  (= address (ethereum/address->checksum address)))
```

Comment:
could you please elaborate on this change?

Reply:
`ethereum/address->checksum` i.e. `toChecksumAddress` converts an arbitrary case address to one with correct checksum case.

So, if the address had the correct checkum case, the result will be the same as input.

Now that you mention this, I also could've used `checkAddressChecksum` which probably does the same thing.

Reply:
Oh well, seems like `checkAddressChecksum` isn't available in our version of web3. 
Never mind, I'll keep it like that. I'll just move `address->checksum` to the `eip55` namespace to keep them together,